deployments/deploy-ssh.sh

```diff
@@ -34,7 +34,7 @@ if [ -z "${SPEC_DIR}" ]; then
 fi
 
 if [ -z "${SCHEDULER}" ]; then
-    SCHEDULER="k8s"
+    SCHEDULER="rke"
```

Comment:
@snigdha-px is the validation after restore that NS is placed in specified project done in next automation backlog ?

Reply:
Added: ValidateProjectOfNamespac